js/languages/en_US.json

```diff
@@ -4261,4 +4262,5 @@
         "zu": "Zulu",
         "zu-ZA": "Zulu (South Africa)"
     }
+    "EUROPEAN_UNION": "European Union",
```

Comment:
@jashot7 Something got borked here. This translation is in a random place. You sure the master you merged from is correct (it seemed to have an odd name)?